### ISSUE TITLE
Settings verbiage for HTTP Basic Auth is inaccurate

### ISSUE THREAD

**Issue by systemoperator** [keepassxreboot/keepassxc#4619]
[TIP]:  # ( Provide a general summary of the feature in the title above ^^ )
[TIP]:  # ( DO NOT include screenshots of your actual database! )

## Summary
[NOTE]: # ( Provide a brief overview of what the new feature is all about )

HTTP Basic Auth dialogs should be improved. Currently, the first suitable pair of credentials is used and then they get auto-submitted.

## Desired Behavior
[NOTE]: # ( Tell us how the new feature should work, be specific )

If several different credentials are stored, they should be selectable and then submitted. Maybe the whole auto-submission could be removed as well and the user can submit it.

## Context
[NOTE]: # ( Why does this feature matter to you? What unique circumstances do you have? )

When one has different credentials for a site, then always the first one is used, even if different credentials would be preferred for the login.

**Comment by droidmonkey** [comment]
What version of KeePassXC are you using?

**Comment by systemoperator** [comment]
KeePassXC v. 2.5.4.

Oh, but I have tested it now and it works as expected. :)

I have just read the description in the settings, stating:

_"If credentials are found for a page and the login-type is an HTTP Basic Auth request, KeePassXC-Browser tries to login **with the first given credentials.**"_

This is actually not true anymore. Because one can already choose the appropriate credentials from the list, if there are more than one present. That's marvelous!

I have deliberately not used this feature for a long time, because I did not like the behavior how it described in the settings, due to the fact, that I have more than one pair of credentials and I want to choose the approriate one. In my opinion, the description should be adapted, stating that one will then be prompted to choose the right pair of credentials, if there are more than one.

If only one pair of credentials is saved, will those then be used and currently **automatically** submitted? (My emphasis lies on _automatically_.)
